end_to_end/nlp_mlops_company_sentiment/scripts/data-processing.py

```diff
@@ -11,8 +11,14 @@
 from smjsindustry.finance.processor_config import EDGARDataSetConfig
 
 
+
+
 if __name__=='__main__':
     
+    # register to get a newscatcher api key : https://newsapi.org/register
+    
+    X_API_KEY ='INSERT YOUR NEWSCATCHER API KEY HERE' # Copy your newscatcher API Key HERE
```

Comment:
Style nit - add brackets around fields the user should update, ie `'<INSERT YOUR NEWSCATCHER API KEY HERE>'`